sbb_newspapers_org_image/eynollah.py

```diff
@@ -1,306 +1,198 @@
-#! /usr/bin/env python3
-
-__version__ = '1.0'
+"""
+tool to extract table form data from alto xml data
```

Comment:
DocString should be updated and start with a capital letter and end with a fullstop.

Reply:
+1 and offer more details. And we should reuse that in the docstring for the CLI and upcoming OCR-D CLI.